src/tf/parse_generic_op.cpp

```diff
@@ -50,7 +50,8 @@ struct parse_generic_op : op_parser<parse_generic_op>
                           const tf_parser::node_info& info,
                           const std::vector<instruction_ref>& args) const
     {
-        return info.add_instruction(make_op(opd.op_name), args);
+        const auto& op = make_op(opd.op_name);
```

Comment:
I think this is simpler to understand as the previous code since this is just an add instruction.